ajax/libs/thrift/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "thrift",
+  "filename": "thrift.min.js",
+  "description": "The Apache Thrift software framework, for scalable cross-language services development, combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages.",
```

Comment:
This sentence is from homepage, which is different from `package.json/description` in the upstream. Not sure if we can or should change it.